### PR TITLE
feature/create scripts to support the compilation of the project

### PR DESCRIPTION
**Pull request by McBlankenburg**

### CODE REVIEW

File: scripts/simple-as-coding-database/simple-as-coding-db-docker-linux.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+# Starting a new container
+docker run -d \
+--name $CONTAINER_NAME \
+-e POSTGRES_USER=postgres \
+-e POSTGRES_PASSWORD=root \
+-e POSTGRES_DB=simpleascoding \
+-p 5432:5432 \
+postgres
+echo "Created new container: $CONTAINER_NAME"
```
Comment (dawciobiel) [inline scripts/simple-as-coding-database/simple-as-coding-db-docker-linux.sh:19]: What if the command:
`docker run ...` 
ends unsuccessfully?
Message:
`echo "Created new container: $CONTAINER_NAME"`
will be wrong!

Maybe use conditional mode to display the message?

### Links
- https://unix.stackexchange.com/questions/22726/how-to-conditionally-do-something-if-a-command-succeeded-or-failed
- https://ss64.com/nt/if.html

File: scripts/simple-as-coding-database/simple-as-coding-db-docker-windows.cmd [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+REM Starting a new container
+docker run -d ^
+--name %CONTAINER_NAME% ^
+-e POSTGRES_USER=postgres ^
+-e POSTGRES_PASSWORD=root ^
+-e POSTGRES_DB=simpleascoding ^
+-p 5432:5432 ^
+postgres
+
+echo Created new container: %CONTAINER_NAME%
```
Comment (dawciobiel) [inline scripts/simple-as-coding-database/simple-as-coding-db-docker-windows.cmd:21]: Same as above.
Maybe use conditional mode to display the message?

### Links
- https://unix.stackexchange.com/questions/22726/how-to-conditionally-do-something-if-a-command-succeeded-or-failed
- https://ss64.com/nt/if.html

File: scripts/simple-as-coding-database/simple-as-coding-db-docker-windows.cmd [hunk 3]
```diff
@@ -0,0 +1,21 @@
+@echo off
+set CONTAINER_NAME=simpleascoding
+
+REM Check if the container already exists, if so, delete it
```
Comment (dawciobiel) [inline scripts/simple-as-coding-database/simple-as-coding-db-docker-windows.cmd:4]:
```suggestion
rem Check if the container already exists, if so, delete it
```

File: scripts/simple-as-coding-database/simple-as-coding-db-docker-windows.cmd [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+docker ps -a --format "{{.Names}}" | findstr /r /c:"^%CONTAINER_NAME%$" 2>nul
+if %errorlevel% equ 0 (
+    echo Deleting old container: %CONTAINER_NAME%
+    docker stop %CONTAINER_NAME% > nul 2>&1
+    docker rm %CONTAINER_NAME% > nul 2>&1
+)
+
+REM Starting a new container
```
Comment (dawciobiel) [inline scripts/simple-as-coding-database/simple-as-coding-db-docker-windows.cmd:12]:
```suggestion
rem Starting a new container
```

**Review by dawciobiel — CHANGES_REQUESTED** [review]
Request changes

**Review by dawciobiel — CHANGES_REQUESTED** [review]
Request changes

File: README.md [hunk 5]
```diff
@@ -31,11 +31,57 @@ Platform for teachers &amp; students, to offer or search for tutoring.
...
+```shell
+docker
+```
+wait for docker daemon in docker desktop to start before running the script below
+```shell
+call scripts/simple-as-coding-database/simple-as-coding-db-docker-windows.cmd
+```
+#### 1.2. Linux:
+```shell
+sudo systemctl start docker
```
[the remark this answers is not in the document]
Comment (McBlankenburg) [reply README.md:59]: valuable note, thanks :)

File: scripts/simple-as-coding-database/simple-as-coding-db-docker-linux.sh [hunk 6]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```
[the remark this answers is not in the document]
Comment (McBlankenburg) [reply scripts/simple-as-coding-database/simple-as-coding-db-docker-linux.sh:1]: at the moment I didn't see the need to use bash's functionality, but all in all, it's better to use a newer standard.

File: scripts/simple-as-coding-database/simple-as-coding-db-docker-linux.sh (hunk 1 above)
Comment (McBlankenburg) [reply scripts/simple-as-coding-database/simple-as-coding-db-docker-linux.sh:19]: True! I supplemented the script with displaying several types of errors.

**Comment by McBlankenburg** [comment]
I have issued a new pull request

**Pull request closed without merging**